src/pages/Detail/component/DetailIntro.jsx

```diff
@@ -0,0 +1,232 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import styled from 'styled-components';
+import Modal from 'react-modal/lib/components/Modal';
+import Creator from './Creator';
+import Review from './Review';
+import ReviewModal from './ReviewModal';
+import { AiOutlineClose } from 'react-icons/ai';
+
+function DetailIntro({
+  image,
+  classTitle,
+  classIntroduce,
+  creatorName,
+  profile_image,
+}) {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [reviews, setReviews] = useState([]);
+
+  const showModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const closeModal = useCallback(() => {
+    setIsModalOpen(false);
+  }, []);
+
+  useEffect(() => {
+    fetch(`http://localhost:3000/data/data_review.json`)
+      .then(response => response.json())
+      .then(result => setReviews(result.data));
+  }, []);
+
+  if (!reviews) return <>loading...</>;
+
+  return (
+    <>
+      <InfoTitle>클래스 정보</InfoTitle>
+      <InfoText1>클래스 분량</InfoText1>
+      <InfoText2>5개 챕터, 30개 세부강의</InfoText2>
+      <InfoText1>수강 가능일</InfoText1>
+      <InfoText2>바로 수강 가능</InfoText2>
+      <InfoText1>자막 포함 여부</InfoText1>
+      <InfoText2>포함</InfoText2>
+
+      <Line id="reviewConer" />
+      <ReviewTitle>
+        <Title>실제 수강생 후기</Title>
+        <ReviewBtn onClick={showModal}>후기 작성하기</ReviewBtn>
+      </ReviewTitle>
+
+      <Modal
+        isOpen={isModalOpen}
+        onRequestClose={closeModal}
+        style={modalDisplay}
+      >
+        <XBtn onClick={closeModal}>
+          <AiOutlineClose />
+        </XBtn>
+        <ReviewModal
+          reviews={reviews}
+          setReviews={setReviews}
+          classTitle={classTitle}
+          closeModal={closeModal}
+        />
+      </Modal>
+
+      <ImageWrap>
+        {reviews?.map(item => {
+          return item.image_url === '' ? null : (
+            <ImageContainer key={item.review_id}>
+              <ReviewImg src={item.image_url} alt="후기 이미지" />
+            </ImageContainer>
+          );
+        })}
+      </ImageWrap>
+
+      <div>
+        {reviews?.map(item => {
```

Comment:
```suggestion
        {reviews?.map((review_id, image_url, content}) => {
```
객체에 담긴 값을 사용하는 게 3가지 이상일 경우에는 구조분해 할당해서 사용하는 것이
유지보수 측면에서 좋습니다